packages/checkout-storefront/src/lib/utils/url.ts

```diff
@@ -14,6 +15,13 @@ const queryParamsMap = {
   token: "passwordResetToken",
   email: "passwordResetEmail",
   saleorApiUrl: "saleorApiUrl",
+  // payment flow
+  transaction: "transaction", // allows us to process started transaction
+  processingPayment: "processingPayment", // tell the processing screen to show up
+  // adyen
+  redirectResult: "redirectResult",
+  resultCode: "resultCode",
+  type: "type",
```

Comment:
If those prefixes are only meant for Adyen shouldn't they have a prefix? I mean we will probably add support for more payment providers in the future so I'm wondering if it's ok to reuse those query params or will there be some sort of conflict?

Reply:
I mean this would require us to rewrite those in the adyen app, since these params are added by adyen api. I'm not entirely sure if the dropin itself uses the URL somehow underneath, so I think it's a bit safer to not do that? if the case emerges where we have two payment apps that use the same query params we can always add a `paymentGateway` query param sent to the API, so when the redirect happens we know which one should handle the response. Do you think that'd work?

Reply:
Yup, I think adding a query param to `redirectUrl` would work 👍🏻